shared/actions/fs/index.js

```diff
@@ -332,7 +408,7 @@ function* fsSaga(): Saga.SagaGenerator<any, any> {
   yield Saga.safeTakeEvery(FsGen.filePreviewLoad, filePreview)
   yield Saga.safeTakeEvery(FsGen.favoritesLoad, listFavoritesSaga)
   yield Saga.safeTakeEvery(FsGen.favoriteIgnore, ignoreFavoriteSaga)
-  yield Saga.safeTakeEveryPure(FsGen.mimeTypeLoad, loadMimeType, loadMimeTypeResult)
+  yield Saga.safeTakeEvery(FsGen.mimeTypeLoad, loadMimeType)
```

Comment:
this can still be pure I think, just have `loadMimeType` return whatever it's yielding